themes/default/layouts/community/developer-advocates/single.html

```diff
@@ -3,63 +3,214 @@
 {{ end }}
 
 {{ define "main" }}
+    {{ $pageContext := . }}
+    {{ $devAdvocates := slice "Kat Cosgrove" "David Flanagan" "Laura Santamaria" "Matt Stratton" }}
+
     {{ $personData := index $.Site.Data.team.team .Params.id }}
+    <section class="max-w-5xl container mx-auto my-8 px-4 text-lg font-display">
+        <div class="w-full flex flex-col items-end">
+            <div  class="mb-2">Selected Profile</div>
+            <div class="developer-advocate-nav mb-4">
+                <li class="header-nav-item-drop-down">
+                    <span class="dropdown-title whitespace-nowrap">{{ $personData.name }}</span>
+                    <span class="dropdown-menu-caret"></span>
+                    <div class="header-nav-drop-down-menu">
+                        <ul>
+                            {{ range $fullName := $devAdvocates }}
+                                <li>
+                                    {{ $devAdvocatePage := (printf "/community/developer-advocates/%s" ($fullName | urlize)) }}
```

Comment:
This works now, but could easily break or fall out of sync with `team` data. I'd suggest using the same method we use to build the index page (and both would probably be safer with `relref`s, too): https://github.com/pulumi/pulumi-hugo/blob/e3c2318464d2ce98f1af1d0a5a5e6bc95c084da1/themes/default/layouts/community/developer-advocates/list.html#L19-L21

Reply:
also I could be wrong but with me wanting my displayed name to be "Matty Stratton" but the file is `matt-stratton` that also maybe is weird?